driver_test.go

```diff
@@ -78,6 +92,9 @@ func createDSN(timezone string) {
 	if rolename != "" {
 		parameters.Add("role", rolename)
 	}
+
+	parameters.Add("privateKey", base64.RawStdEncoding.EncodeToString(x509.MarshalPKCS1PrivateKey(TestPrivateKey)))
```

Comment:
We should avoid using PKCS1 private key. We should use PKCS8. You should add another sanity check to reject PKCS1 format of private key if found.